Cargo.toml

```diff
@@ -56,6 +56,8 @@ std = [
     "dep:cc",
     "rand/std",
     "rand/std_rng",
+    "rand/os_rng",
+    "rand/thread_rng",
```

Comment:
Just `thread_rng` would be sufficient because all others are implied by it, but I guess for explicitness we could leave that as-is?